test_dependencies.toml

```diff
@@ -233,7 +233,19 @@ targets = [ "thin", "thin-pool",]
 ["/thin/fs-bench/fio/thin-preallocated"]
 executables = [ "blockdev", "dd", "dmsetup", "echo", "fio", "fsck.ext4", "mkfs.ext4", "mount", "umount",]
 targets = [ "thin", "thin-pool",]
-["/vdo/vdo_tests"]
-executables = [ "blockdev", "dd", "dmsetup", "echo", "fio", "fsck.ext4", "mkfs.ext4", "mount", "umount", 'vdoformat']
+
+["/vdo/creation/create01"]
+executables = [ "blockdev", "dmsetup", "echo", 'vdoformat']
+targets = [ "vdo",]
+
+["/vdo/dedupe/dedupe0"]
+executables = [ "blockdev", "dmsetup", "fio", "echo", 'vdoformat']
```

Comment:
These lists get updated by running the test to reflect the programs actually used. My results show several other programs should be listed, and "fio" doesn't belong in the list.

Reply:
If it gets updated each time the tests are run, why is this file checked in?

Reply:
I assume it's checked in because you need a starting point to check dependencies before running tests. Silently updating them on every run seems a bit much to me, since it means I'll unexpectedly get modified "source" files compared to my repo; I'd prefer to see a warning and an update option or something like that, but whatever.

The list I'm seeing includes udevadm and the gen_dataset.py script in addition to the ones you've listed. I haven't rerun the test for a couple of your updates, though, so I'm not sure that's current.